_posts/2018-10-05-a-macroscopic-take-on-metaprogramming.md

```diff
@@ -0,0 +1,293 @@
+---
+layout: post
+title:  "A macroscopic take on metaprogramming"
+date:   2018-10-05 17:00:00 +0100
+categories: elixir ruby javascript
+author: Natacha Segala
+---
+If you are as keen on DIY as I am and equally as helpless, the first time you're presented with some sort of power tool you may think 'This is scary, how could I possibly use this?'.
+
+In some respects, metaprogramming can feel just like this.
+
+A quick look at [wikipedia's definition](https://en.wikipedia.org/wiki/Metaprogramming){:target= "_blank"} gives us:
```

Comment:
capitalize Wikipedia

~A quick look at~ is superfluous.

Reply:
I like the phrasing personally, but agree about capitalization.

Reply:
Quick look was expressing my "I didn't bother including the whole of the definitions given by Wikipedia"